sdk/mixedreality/mixedreality-authentication/src/tracing.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTracer } from "@azure/core-tracing";
+import { Span, SpanOptions, SpanKind } from "@opentelemetry/api";
+import { OperationOptions } from "@azure/core-http";
+
+type OperationTracingOptions = OperationOptions["tracingOptions"];
+
+/**
+ * Creates a span using the global tracer.
+ * @ignore
```

Comment:
There's been some churn around which tag to use for preventing something from being documented publicly. I believe we've finally settled on just using the `@internal` tag.

Reply:
I just tried switching that one to `@internal` and it actually caused it to start showing up in the documentation. So, if there's churn coming that changes the behavior, should I add both?

Reply:
Ah! I added the `--stripInternal` switch to typedoc and now it's picking up the `@internal` tag as well.